src/audio/detect_test.c

```diff
@@ -64,6 +64,7 @@ struct comp_data {
 	uint32_t detect_preamble; /**< current keyphrase preamble length */
 	uint32_t keyphrase_samples; /**< keyphrase length in samples */
 	uint32_t buf_copy_pos; /**< current copy position for incoming data */
+	uint32_t history_depth;
```

Comment:
some comment here, like "used to inform kpb draining about history buffer size, in bytes".